foundation/web/web.go

```diff
@@ -83,15 +84,18 @@ func (a *App) Handle(method string, group string, path string, handler Handler,
 // to the application server mux.
 func (a *App) handle(method string, group string, path string, handler Handler) {
 	h := func(w http.ResponseWriter, r *http.Request) {
-		ctx, span := AddSpan(r.Context(), "foundation.app.handle")
-		defer span.End()
+		ctx := r.Context()
+		span := trace.SpanFromContext(ctx)
 
-		ctx = SetValues(ctx, &Values{
+		ctx = SetValues(r.Context(), &Values{
```

Comment:
This needs to be `SetValues(ctx`

Reply:
I need @dlsniper to look at this. I think he identified that AddSpan would look at the context. I am not sure.